packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -157,53 +156,43 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
+
         return (
           <Provider
             {...provider}
-            UNSAFE_className={classNames(listStyles, 'react-spectrum-ListViewItem', 'is-dragging')}
-            UNSAFE_style={{width: itemWidth, paddingInlineStart: 0}}>
-            <div className={listStyles['react-spectrum-ListViewItem-grid']}>
-              <div className={listStyles['react-spectrum-ListViewItem-draghandle-container']}>
-                <div className={listStyles['react-spectrum-ListViewItem-draghandle-button']}>
-                  <ListGripper />
-                </div>
-              </div>
-              {showCheckbox &&
-                <Checkbox
-                  isSelected={isSelected}
-                  UNSAFE_className={listStyles['react-spectrum-ListViewItem-checkbox']}
-                  isEmphasized
-                  aria-label={checkboxProps['aria-label']} />
-              }
-              <SlotProvider
-                slots={{
-                  content: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-                  text: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-                  description: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-description']},
-                  icon: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-icon'], size: 'M'},
-                  image: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-image']},
-                  link: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content'], isQuiet: true},
-                  actionButton: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'], isQuiet: true},
-                  actionGroup: {
-                    UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'],
-                    isQuiet: true,
-                    density: 'compact'
-                  },
-                  actionMenu: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actionmenu'], isQuiet: true}
-                }}>
-                {typeof item.rendered === 'string' ? <Content>{item.rendered}</Content> : item.rendered}
-              </SlotProvider>
+            UNSAFE_style={{width: 194, direction}}>
+            <div className={classNames(listStyles, 'react-spectrum-ListViewItem', 'react-spectrum-ListViewItem-dragPreview', {'react-spectrum-ListViewItem-dragPreview--multiple': isDraggingMultiple})}>
+              <Grid UNSAFE_className={listStyles['react-spectrum-ListViewItem-grid']}>
+                <SlotProvider
+                  slots={{
+                    content: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
+                    text: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
+                    description: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-description']},
+                    icon: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-icon'], size: 'M'},
```

Comment:
small nit: The icon in the drag preview is placed rather close to the text content in RTL locales. It has the proper spacing in LTR locales though

Reply:
Looks like  `padding-inline-end: var(--spectrum-global-dimension-size-100)` is showing up on the right side with RTL for the preview. Looking into why.